.devcontainer/docker-compose.yml

```diff
@@ -5,11 +5,6 @@ services:
     build:
       context: ..
       dockerfile: .devcontainer/Dockerfile
-      args:
-        USER_UID: 1000
-        USER_GID: 1000
-    environment:
-       BUILD_ENGINE: postgresql://postgres:postgres@localhost:5432/postgres
```

Comment:
User ids set in dockerfile, which from some googling matches some microsoft dev container dockerfile template. Figure I'll leave there

Reply:
Build engine will continue to handle like we do the rest of .env stuff